src/pages/Doeun/Main/Components/Article/Article.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import CommentBar from '../CommentBar/CommentBar';
+import UnderBar from '../UnderBar/UnderBar';
+import Comment from '../Comment/Comment';
+import './Article.scss';
+
+class Article extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentList: [],
+      commentValue: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/Doeun/commentData.json', {
+      method: 'GET',
+    })
```

Comment:
- localhost:3000 이라고 명시해준 부분은 생략해주세요. 만약 포트 번호가 변하게 되는 경우, 일일이 fetch 함수를 찾아서 포트 번호를 수정해주어야 하는데, 생략을 하고 작성을 하시면 포트번호가 자동으로 들어가서 그러지 않아도 됩니다.
- fetch 함수의 기본 메서드는 get 메서드 이기 때문에, get 메서드로 http 통신을 하실 경우, option 부분을 생략해주실 수 있습니다.

```suggestion
    fetch('/data/Doeun/commentData.json')
```

Reply:
아하~~~ 수정했습니다!